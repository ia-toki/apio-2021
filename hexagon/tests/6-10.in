4279751dd9ece9095f11fb277ecbef2e68be1cdf
607 722136587 284230913
6 541
1 405
2 436
3 510
4 453
5 404
6 557
1 416
2 454
3 524
4 461
5 418
6 574
1 419
2 469
3 535
4 470
5 427
6 587
1 427
2 482
3 545
4 484
5 436
6 604
1 445
2 493
3 555
4 507
5 446
6 610
1 466
2 502
3 564
4 521
5 453
6 634
1 462
2 527
3 571
4 539
5 466
6 649
1 478
2 543
3 581
4 558
5 479
6 654
1 507
2 534
3 615
4 554
5 503
6 657
1 530
2 536
3 645
4 554
5 522
6 675
1 539
2 552
3 662
4 568
5 527
6 701
1 538
2 564
3 680
4 566
5 549
6 716
1 547
2 572
3 708
4 563
5 564
6 735
1 545
2 598
3 707
4 585
5 579
6 737
1 567
2 612
3 710
4 604
5 590
6 755
5 10
4 1
3 763
2 596
1 616
6 717
5 620
4 573
3 747
2 584
1 591
6 712
5 602
4 554
3 739
2 569
1 572
6 708
5 583
4 546
3 720
2 558
1 564
6 692
5 568
4 539
3 709
2 534
1 566
6 667
5 563
4 531
3 694
2 522
1 566
6 645
5 552
4 532
3 666
2 512
1 554
6 631
5 533
4 516
3 657
2 487
1 557
6 596
5 537
4 489
3 651
2 474
1 539
6 585
5 531
4 471
3 640
2 466
1 525
6 571
5 512
4 468
3 616
2 451
1 517
6 555
5 496
4 458
3 610
2 432
1 502
6 546
5 489
4 433
3 593
2 438
1 470
6 542
5 473
4 427
3 580
2 421
1 465
6 534
5 457
4 422
3 562
2 415
1 459
6 515
5 451
4 404
3 554
2 390
1 449
6 495
5 430
4 396
3 531
2 387
1 425
6 496
5 409
4 393
3 518
2 366
1 428
6 467
5 408
4 376
3 504
2 357
1 417
6 460
5 395
4 365
3 493
2 353
1 394
6 451
5 386
4 345
3 493
2 322
1 395
6 431
5 373
4 325
3 480
2 321
1 369
6 432
5 358
4 318
3 471
2 314
1 348
6 434
5 337
4 314
3 458
2 304
1 338
6 416
5 338
4 293
3 450
2 293
1 329
6 396
5 333
4 274
3 443
2 269
1 323
6 379
5 318
4 264
3 424
2 266
1 297
6 379
5 298
4 258
3 400
2 258
1 285
6 358
5 283
4 252
3 376
2 257
1 262
6 354
5 265
4 246
3 350
2 252
1 251
6 342
5 242
4 247
3 329
2 247
1 228
6 332
5 227
4 242
3 315
2 226
1 225
6 316
5 209
4 229
3 292
2 225
1 196
6 303
5 207
4 204
3 283
2 209
1 197
6 271
5 212
4 186
3 272
2 193
1 194
6 256
5 197
4 176
3 268
2 179
1 179
6 251
5 175
4 172
3 252
2 157
1 182
6 225
5 173
4 158
3 236
2 149
1 169
6 208
5 154
4 147
3 211
2 144
1 148
6 196
5 149
4 127
3 205
2 120
1 140
6 178
5 137
4 113
3 192
2 109
1 128
6 165
5 118
4 112
3 165
2 103
1 114
6 145
5 108
4 98
3 150
2 90
1 105
6 126
5 97
4 90
3 129
2 82
1 96
6 101
5 100
4 74
3 108
2 83
1 78
6 100
5 77
4 71
3 95
2 67
1 67
6 89
5 59
4 67
3 75
2 61
1 55
6 73
5 50
4 57
3 58
2 49
1 49
6 56
5 37
4 41
3 48
2 33
1 24
6 46
5 23
4 27
3 28
2 23
1 13
6 28
5 12
4 13
3 16
2 8
1 1
5 3
6 13
1 5
2 5
3 18
4 3
5 20
6 19
1 21
2 15
3 40
4 17
5 26
6 37
1 33
2 32
3 50
4 33
5 45
6 56
1 42
2 52
3 63
4 57
5 54
6 66
1 63
2 55
3 81
4 53
5 74
6 83
1 67
2 77
3 86
4 82
5 72
6 103
1 73
2 95
3 93
4 93
5 83
6 118
1 80
2 102
3 111
4 99
5 90
6 132
1 98
2 103
3 135
4 113
5 96
6 160
1 107
2 116
3 154
4 126
5 108
6 176
1 116
2 128
3 166
4 143
5 104
6 203
1 116
2 144
3 183
4 148
5 126
6 211
1 134
2 150
3 200
4 156
5 150
6 220
1 153
2 164
3 222
4 164
5 163
6 240
1 167
2 172
3 239
4 183
5 161
6 266
1 167
2 193
3 246
4 190
5 183
6 275
1 177
2 203
3 266
4 193
5 201
6 275
1 197
2 208
3 289
4 194
5 217
6 290
1 213
2 210
3 310
4 210
5 231
6 301
1 237
2 220
3 323
4 228
5 237
6 323
1 241
2 240
3 333
4 243
5 244
6 345
1 244
2 252
3 353
4 252
5 256
6 369
1 242
2 283
3 355
4 276
5 258
6 389
1 261
2 283
3 376
4 290
5 262
6 413
1 266
2 302
3 387
4 307
5 273
6 431
1 277
2 318
3 395
4 321
5 287
6 445
1 286
2 338
3 411
4 330
5 306
6 448
1 307
2 337
3 427
4 341
5 310
6 468
1 313
2 347
3 437
4 355
5 318
6 477
1 321
2 369
3 429
4 386
5 322
6 487
1 338
2 374
3 447
4 391
5 338
6 493
1 359
2 387
3 459
4 407
5 357
6 497
1 374
2 403
3 468
4 421
5 362
6 514
1 385
2 409
3 480
4 429
5 378
6 521
1 398
2 421
3 493
4 440
5 392
