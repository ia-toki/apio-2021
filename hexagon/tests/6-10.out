0a7385ba87a80c36d1403904d6d65e155ddd6e9c
OK
323345347
